test/common/router/router_test.cc

```diff
@@ -90,6 +90,9 @@ class RouterTestBase : public testing::Test {
     router_.downstream_connection_.local_address_ = host_address_;
     router_.downstream_connection_.remote_address_ =
         Network::Utility::parseInternetAddressAndPort("1.2.3.4:80");
+
+    // make the "system time" non-zero, because 0 is considered invalid by DateUtil
```

Comment:
Though super nitty mind
make the "system time" non-zero, because 0 is considered invalid by DateUtil 
->
Make the "system time" non-zero, because 0 is considered invalid by DateUtil.